packages/go/analysis/ad/post.go

```diff
@@ -50,6 +50,9 @@ func PostProcessedRelationships() []graph.Kind {
 		ad.ADCSESC5,
 		ad.ADCSESC6,
 		ad.ADCSESC7,
+		ad.EnrollOnBehalfOf,
+		ad.IssuedSignedBy,
+		ad.NTAuthStoreFor,
```

Comment:
IssuedSignedBy is already in this list above. I believe NTAuthStoreFor is an ingested edge not a post-processed edge.